### DISCUSSION TITLE
Do this model supports timestamps?

### DISCUSSION THREAD

**Discussion by mukhtharcm** [idiap/coqui-ai-TTS#186] [Q&A] [answered]
Do this model support timestamps like http://elevenlabs.io/docs/api-reference/text-to-sp…

**Comment by eginhard** [comment, marked as answer]
~Coqui has a lot of models, you didn't say which one you mean with "this model". But currently none of them support it directly.~

UPDATE: From version 0.27.3, the Synthesizer class can return sentence-level timestamps: https://coqui-tts.readthedocs.io/en/latest/main_classes/synthesizer.html
